### PR TITLE
update documentation and CI dependencies

### PR DESCRIPTION
**Pull request by wkloucek**
Fixes https://github.com/owncloud/ocis/issues/2077

### CODE REVIEW

**Comment by C0rby** [comment]
I think the tests fail because since the config handling was refactored the flag `PROXY_CONFIG_FILE` is only working when you are starting the proxy on it's own otherwise it's being ignored.

File: docs/extensions/hello/building.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ yarn install
 yarn build
 {{< / highlight >}}
 
-The above commands will install the required build dependencies and build the whole frontend bundle. This bundle will we embeded into the binary later on.
+The above commands will install the required build dependencies and build the whole frontend bundle. This bundle will we embedded into the binary later on.
```
Comment (C0rby) [inline docs/extensions/hello/building.md:28]: Here is a typo: `bundle will we embedded` -> `will be embedded`

File: .bingo/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+*   Run `bingo get` to install all tools having each own module file in this directory.
+*   Run `bingo get <tool>` to install <tool> that have own module file in this directory.
+*   For Makefile: Make sure to put `include .bingo/Variables.mk` in your Makefile, then use $(<upper case tool name>) variable where <tool> is the .bingo/<tool>.mod.
+*   For shell: Run `source .bingo/variables.env` to source all environment variable for each tool.
+*   For go: Import `.bingo/variables.go` to for variable names.
+*   See https://github.com/bwplotka/bingo or -h on how to add, remove or change binaries dependencies.
+
+## Requirements
+
+*   Go 1.14+
```
Comment (C0rby) [inline .bingo/README.md:14]: Is this version correct?
The `*.mod` files in `.bingo` state `go 1.16`

**Review by C0rby — APPROVED** [review]
Looks good to me. :+1:

File: .bingo/README.md (hunk 2 above)
Comment (wkloucek) [reply .bingo/README.md:14]: Bingo itself works from Go 1.14 on. 

The `*.mod` files in `.bingo`  contain the Go version of the creation / last update. But still should work with Go 1.14 though :thinking:

**Pull request merged**